analysis/01_data_processing.Rmd

```diff
@@ -1,703 +1,404 @@
 ```
 
-```{r dpz long tailed macaques}
-
-dpz_mac_raw <- read_excel("../data/raw_data/ManyPrimates_data dpz_long-tailed macaques.xlsx", 1)
-# check column names
-names(sample) %in%  names(dpz_mac_raw)
-# check condition names
-unique(sample$condition) %in%  unique(dpz_mac_raw$condition)
-
-# check species
-unique(dpz_mac_raw$species)
-# check subject
-unique(dpz_mac_raw$subject_site)
-# check age
-unique(dpz_mac_raw$age)
-# check cup distance
-unique(dpz_mac_raw$cup_distance)
-# check task experience
-unique(dpz_mac_raw$task_experience) %in% unique(sample$task_experience)
-# check board size
-unique(dpz_mac_raw$board_size)
-# check test situation
-unique(dpz_mac_raw$test_situation) %in% unique(sample$test_situation)
-# check session
-unique(dpz_mac_raw$session)
-# check block
-unique(dpz_mac_raw$block)
-# check trial
-unique(dpz_mac_raw$trial)
-# check pick
-unique(dpz_mac_raw$pick)
-# check hiding locations
-unique(dpz_mac_raw$hiding_location) %in% unique(sample$hiding_location)
-# check correct
-unique(dpz_mac_raw$correct)
+```{r}
+doChecks(dpz_mac_raw)
+```
 
+```{r}
 # change data file
 dpz_mac <- dpz_mac_raw %>%
-  mutate(species = recode_factor(species,"long-tailed macaque" = "long_tailed_macaque"),
+  mutate(site = toupper(site),
+         species = gsub('[ -]', '_', species),
          correct = as.numeric(correct),
-         secondary_pick = as.character(secondary_pick))%>%
-  filter(!is.na(pick))
+         secondary_pick = as.character(secondary_pick))
 ```
 
-```{r edinburgh squirel monkeys}
-
-edin_squ_raw <- read_excel("../data/raw_data/ManyPrimates Pilot Coding_Squirrel monkeys_EW_17.7.18.xlsx", 1)
-# check column names
-names(sample) %in%  names(edin_squ_raw)
-# check condition names
-unique(sample$condition) %in%  unique(edin_squ_raw$condition)
-
-# check species
-unique(edin_squ_raw$species)
-# check subject
-unique(edin_squ_raw$subject_site)
-# check age
-unique(edin_squ_raw$age)
-# check cup distance
-unique(edin_squ_raw$cup_distance)
-# check task experience
-unique(edin_squ_raw$task_experience) %in% unique(sample$task_experience)
-# check board size
-unique(edin_squ_raw$board_size)
-# check test situation
-unique(edin_squ_raw$test_situation) %in% unique(sample$test_situation)
-# check session
-unique(edin_squ_raw$session)
-# check block
-unique(edin_squ_raw$block)
-# check trial
-unique(edin_squ_raw$trial)
-# check pick
-unique(edin_squ_raw$pick)
-# check hiding locations
-unique(edin_squ_raw$hiding_location) %in% unique(sample$hiding_location)
-# check correct
-unique(edin_squ_raw$correct)
+## Lincoln Park Zoo
 
+```{r}
+linc_raw <- read_excel(str_c(path, "LPZ 3 cups data June 2018 Hopper Egelkamp Ross.xlsx"))
+```
+
+```{r}
+doChecks(linc_raw)
+```
+
+```{r}
 # change data file
-edin_squ <- edin_squ_raw %>%
-  filter(!is.na(pick))
+linc <- linc_raw %>%
+  mutate(species = tolower(species),
+         board_size = 61.5)
 ```
 
+> **!!!** Where does the new board_size number (61.5) come from? --jw
```

Comment:
convert inch to cm

Reply:
@manuelbohn So that's what I thought at first but 35.5 inches = 90 cm...

Re: where to put it, I'll make a comment directly in the code before merging (same as for the ACCI board size

Reply:
Ok, my memory really did not serve me well here. I went through my emails again and found an exchange with Lydia in which I asked her if the board size was correct (35 cm seemed to small given the cup distance). She replied that 35 cm was the depth and not the width of the board. the 61.5cm is the value she gave me for the width. Sorry for the confusion!